todo_app/test/test_appUiTest.py

```diff
@@ -38,7 +38,7 @@ def delete_trello_board(board_id):
 
 @pytest.fixture(scope="module")
 def driver():
- with webdriver.Firefox() as driver:
+ with webdriver.Chrome() as driver:
```

Comment:
Chromedriver is currently failing to start, adding the extra options mentioned in the exercise should help with that:
```suggestion
    opts = webdriver.ChromeOptions()
    opts.add_argument('--headless')
    opts.add_argument('--no-sandbox')
    opts.add_argument('--disable-dev-shm-usage')
    with webdriver.Chrome("./chromedriver", options=opts) as driver:
```

Reply:
acted on it